config.cmake

```diff
@@ -599,3 +472,135 @@ config_option(
 )
 
 add_config_library(kernel "${configure_string}")
```

Comment:
This line has to be at the bottom because there's configs defined in `include(src/drivers/config.cmake)`.

Reply:
Thanks.